fastlane/Fastfile

```diff
@@ -626,7 +655,7 @@ def screenshots_directory()
   File.join(fastlane_directory, "screenshots")
 end
 
-def promo_screenshots_directory()
+def promo_screenshots_directory
```

Comment:
I super like that this is extracted, but IMHO it should be moved to the release toolkit and used as a default anywhere we'd reference the value. It might not even need to be customizable at this point...we're pretty consistent with it.

(the end goal being to make the fastfiles very small and share as much as is reasonable without over abstracting 😄

Reply:
> (the end goal being to make the fastfiles very small and share as much as is reasonable without over abstracting 😄

Nice. I like it, it makes sense to me to extract with reasonable defaults and then add the ability to override where needed.